packages/sitecore-jss-proxy/src/middleware/editing/index.test.ts

```diff
@@ -198,27 +172,390 @@ describe('editingRouter', () => {
             },
             path: '/foo/config',
           },
+          render: renderConfig,
         })
       );
 
       request(app)
         .get('/api/editing/foo/config')
         .query({ secret: 'correct' })
-        .expect(200)
-        .end((err, res) => {
-          if (err) return done(err);
-          expect(res.body).to.deep.equal({
+        .expect(
+          200,
+          {
             components: ['component1'],
             packages: {
               foo: '1.0.0',
             },
             editMode: 'metadata',
-          });
+          },
+          done
+        );
+    });
+  });
+
+  describe('/render', () => {
+    const requiredParams = ['sc_site', 'sc_itemid', 'sc_lang', 'route', 'mode'];
+
+    const validQS: RenderMetadataQueryParams = {
+      secret: 'correct',
+      sc_site: 'site',
+      sc_itemid: '{Guid}',
+      sc_lang: 'en',
+      sc_layoutKind: LayoutKind.Shared,
```

Comment:
I added it here just in order to provide a custom layoutKind, the value that is here doesn't affect the final result since I stub the "fetchEditingData" method